src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  if (sourceString) {
+    return sourceString
+      .split(';')
+      .map((row) => row.split(':').map((param) => param.trim()))
+      .filter((row) => row.length > 1)
+      .map((e) => ({ [e[0]]: e[1] }))
+      .reduce((prew = {}, e) => ({ ...prew, ...e }));
```

Comment:
The default parameter assignment `prew = {}` in the `reduce` method is redundant. The accumulator `prew` will be initialized to the first element of the array by default if no initial value is provided, or to the initial value if one is provided. Since you are using an empty object `{}` as the initial value, you can safely remove the default assignment.